nesta/core/tasks/projects/ai_diversity/doc2cluster.py

```diff
@@ -0,0 +1,227 @@
+"""
+Fetches articles from DB and transforms variable-length texts (usually from sentences to paragraphs)
```

Comment:
[Style] Just add the title at the top here, as in:

`Doc2Cluster`
`===========`